obspy/imaging/scripts/mopad.py

```diff
@@ -772,13 +776,19 @@ def _find_faultplanes(self):
         """
         # reference Double Couple (in NED basis)
         # it has strike, dip, slip-rake = 0,0,0
-        refDC = np.matrix([[0., 0., -1.], [0., 0., 0.], [-1., 0., 0.]],
-                          dtype=np.float)
+        refDC = np.array([
+            [0, 0, -1],
+            [0, 0, 0],
+            [-1, 0, 0],
+        ], dtype=float)
```

Comment:
Same comment here, it may be a bit more robust to use a specific np float type here.